submissions/lisnyak-o/popup-kottans/index.html

```diff
@@ -0,0 +1,137 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <meta name="HandheldFriendly" content="true">
+  <meta name="MobileOptimized" content="width">
+  <meta name="apple-mobile-web-app-capable" content="yes">
+  <meta http-equiv="X-UA-Compatible" content="ie=edge">
+  <link rel="shortcut icon" href="favicon.png" type="image/x-icon">
+  <link
+    href="https://fonts.googleapis.com/css?family=Fira+Sans:100,100italic,200,200italic,300,300italic,regular,italic,500,500italic,600,600italic,700,700italic,800,800italic,900,900italic"
+    rel="stylesheet" />
```

Comment:
In my opinion, load more styles for font and not use them it is unreasonable

Reply:
I don't  understand why do you think so, but I use it's fonts. 
For example, correct me if I don't understand you correctly. Thanks.
![font](https://user-images.githubusercontent.com/20680156/96986189-fdc01b00-1529-11eb-98a0-1b86c0dc7784.png)
Do you mean about ",italic,500,500italic,600,600italic,700,700italic,800,800italic,900,900italic" ?

Reply:
You use only one style `font-weight: 400`